src/reconstruct/reconstruction.cpp

```diff
@@ -33,15 +33,6 @@
 
 namespace parthenon {
 
-namespace {
-
-// TODO(felker): replace these hand-rolled linear algebra routines with a real library
-constexpr Real lu_tol = 3e-16;
-int DoolittleLUPDecompose(Real **a, int n, int *pivot);
-void DoolittleLUPSolve(Real **lu, int *pivot, Real *b, int n, Real *x);
```

Comment:
I think it's fine to remove this. But just to check, @pgrete @jmstone is there a reason to keep these around?

Reply:
I think  this can go (and being reintroduced later if required).